coredns/Acornfile

```diff
@@ -14,9 +17,14 @@ containers: {
         scale: args.scale
         ports: "53/udp"
         image: "docker.io/coredns/coredns:\(args.version)"
-        if args.corefile != "" {
-            files: "./Corefile": args.corefile
+        
+        files: {
+            if args.corefile != "" {
+                "./Corefile": args.corefile
```

Comment:
Where is this relative to? Probably would be clearer if it was an explicit path.

Reply:
This is relative to where the `coredns` binary lives. Since the image is static though, you're right in that we can be more explicit about where this is living.